.travis.yml

```diff
@@ -1,25 +1,33 @@
 # This provides us with GCC binary support so we can run bash
+dist: trusty
```

Comment:
without that the build will fail as they have changed things in ubuntu with sudo and its a build env. so no problem